### PR TITLE
Theme basepath

### LINKED ISSUE #1970: Keep basepath while theme loading
In `loadThemeConfig` the basepath should be keep as relative path and which should also saved into database.

https://github.com/luyadev/luya/blob/849532964e50b96969069adca01d884d9f8ad5e8/core/theme/ThemeManager.php#L57

### PR DESCRIPTION
**Pull request by boehsermoe**
Problem here is, that the imported base path (the theme name) will be stored in the database. And the same base path will be searching in the database. But if the base path is changed while config loading from alias to an absolute path, this theme cannot found.

this closes #1970

### CODE REVIEW

File: core/theme/ThemeManager.php [hunk 1]
```diff
@@ -50,12 +50,14 @@ public function loadThemeConfig(string $basePath)
...
             $dir = Yii::getAlias($basePath);
-        } elseif (strpos($basePath, '/') !== 0) {
-            $dir = $basePath = Yii::$app->basePath . DIRECTORY_SEPARATOR . $basePath;
-        } else {
+        } elseif (strpos($basePath, '/') === 0) {
+            // absolute path
             $dir = $basePath;
+        } else {
+            // relative path
+            $dir = Yii::$app->basePath . DIRECTORY_SEPARATOR . $basePath;
```
Comment (nadar) [inline core/theme/ThemeManager.php]: Would be nice to have a test about relative paths.

**Comment by boehsermoe** [comment]
I have found one more problem with relative and absolute path. Theme base path will be stored in the database and have to be found also if the app is installed in other location or maschine.

My idea was to save all theme base paths as alias. Each loaded theme from a package will be imported and stored with the alias "@vendor" in the db. https://github.com/luyadev/luya/pull/2002/files#diff-6bb382275a22b5600c36a95ae7d0f1dbR185

Problem could be, that already exists themes would be imported again with the alias base path and the active theme (with the old basepath) cannot found. I have to test this behaviour.

**Comment by nadar** [comment]
Sorry i am a bit outdated, but what is the situation with this PR, does it fix the issue (https://github.com/luyadev/luya/pull/2002#issuecomment-601024034) or does it conflict with your comment. can we merge? Current version would be 1.2 (please adjust changelog)

**Comment by boehsermoe** [comment]
This PR should be ready to merge. Every theme from a vendor package will be imported again with a new base path and the old entry will be delete from the importer automatically.

**Pull request merged**